assets/js/util.js

```diff
@@ -120,7 +120,10 @@ $(document).ready(function () {
     resizeFooter();
     $(window)
         .on('hashchange', persistChoices)
-        .resize(resizeFooter);
+        .resize(function () {
+            resizeFooter();
+            populateTranslationList();
```

Comment:
`populateTranslationList` is a pretty expensive function. I don't think we should run it on every `resize` event. Can you look into debouncing and find a drop-in function that would help us out?